PltApp.H

```diff
@@ -348,7 +349,7 @@ class AVXGrab {
   public:
     AVXGrab(Display *display) : bIsGrabbed(true), cachedDisplay(display) {
       XSync(cachedDisplay, False);
-      XGrabServer(display);
+      //XGrabServer(display);
```

Comment:
I need to refactor this out into a separate PR. This could be controlled by an ifdef so Amrvis will work with x11vnc or xpra.

Reply:
Is the grab even necessary with modern hardware? As I understand it, the main reasons for grabbing the server are to prevent other windows from trying to redraw themselves while drawing over them (which shouldn't be needed here, since the rectangle is only drawn over our own widgets), and to prevent other apps from congesting the server while doing something interactive (like rubber-banding). I couldn't see any performance differences when I commented out the grab call (running on a local Xorg server). If anything, it was slightly less flickery.